packages/paywall/package.json

```diff
@@ -21,7 +21,8 @@
     "lint": "eslint --resolve-plugins-relative-to ../eslint-config --ext .tsx,.ts,.js src/",
     "test": "NODE_ENV=test vitest run --coverage",
     "clean": "rm -rf build dist coverage",
-    "ci": "yarn test && yarn build"
+    "ci": "yarn test && yarn build",
+    "example": "vite --config example/vite.config.js example/src"
```

Comment:
Move to examples repo

Reply:
I don't think we should move this. It is good to have it in the same folder so can quickly test things for any changes we make.